pipeline/mist.go

```diff
@@ -39,7 +40,15 @@ func (m *mist) HandleStartUploadJob(job *JobInfo) (*HandlerOutput, error) {
 	// Arweave URLs don't support HTTP Range requests and so Mist can't natively handle them for segmenting
 	// This workaround copies the file from Arweave to S3 and then tells Mist to use the S3 URL
 	if clients.IsArweaveOrIPFSURL(job.SourceFile) {
-		newSourceURL, err := inSameDirectory(job.TargetURL, "source", "arweave-source.mp4")
+		if !isVideo(job.RequestID, job.SourceFile) {
```

Comment:
Does it make sense to do this check for all videos (i.e not just Arweave / IPFS ones)?

Reply:
sounds reasonable to me 👍